### PR TITLE
Distinct Numbers python solution

### PR DESCRIPTION
**Pull request by maggie-j-liu**

### CODE REVIEW

**Review by dongliu0426 — APPROVED** [review]

File: solutions/cses-1621.mdx [hunk 1]
```diff
@@ -21,13 +25,13 @@ using namespace std;
 
 int main() {
 	int N; cin >> N;
-	int arr[N];
+	int arr[200000];
```
Comment (SansPapyrus683) [inline solutions/cses-1621.mdx:28]: maybe just replace this with a vector

File: solutions/cses-1621.mdx [hunk 2]
```diff
@@ -66,4 +69,21 @@ public class DistinctNumbers {
...
+<PySection>
+
+```py
+n = int(input())
+# create a sorted list of the numbers
+numbers = sorted(map(int, input().split()))
+ans = 1
+for i in range(1, n):
+	# if the current number is different from the previous
+	# it is a distinct number so we add 1 to the answer
```
Comment (SansPapyrus683) [inline solutions/cses-1621.mdx:81]: if you're gonna add comments to the python sol might as well be consistent & add them to all the sols

File: solutions/cses-1621.mdx [hunk 3]
```diff
@@ -21,18 +25,17 @@ using namespace std;
 
 int main() {
 	int N; cin >> N;
```
Comment (SansPapyrus683) [inline solutions/cses-1621.mdx:27]:
```suggestion
int N;
cin >> N;
```

File: solutions/cses-1621.mdx [hunk 4]
```diff
@@ -59,6 +61,8 @@ public class DistinctNumbers {
 		Arrays.sort(arr);
 		int ans = 1;
 		for (int i = 1; i < n; i++) {
+			// if the current number is different from the previous
+			// it is a distinct number so we add 1 to the answer
```
Comment (SansPapyrus683) [inline solutions/cses-1621.mdx:65]:
```cpp
/*
 * stylize multiline comments
 * like this please
 */
```

**Review by SansPapyrus683 — APPROVED** [review]
yeah seems good enough

**Pull request merged**